public/js/views/management/pay-methods.jsx

```diff
@@ -20,6 +21,10 @@ export default class PayMethods extends Component {
 
   handleDelPayMethod = e => {
     e.preventDefault();
+    if (isDisabled(e.target)) {
+      console.log('Link is disabled. no-op');
```

Comment:
could this be more specific like "not deleting pay methods because clicked button was disabled" ?